R/pkgs/report_generation/inst/rmarkdown/templates/state_report/skeleton/skeleton.Rmd

```diff
@@ -142,14 +142,14 @@ cap <- paste0("**Fig. ", fig_counter, "**: Daily hospital occupancy for 15 simul
 ```{r load_cum_inf_geounit_dates}
 ###Load cumulative infections on specific dates
 doParallel::registerDoParallel(ncores)
-
 cum_inf_dates <- load_cum_inf_geounit_dates(scn_dirs,
-                           num_files = nfiles,
-                           display_dates = config$report$formatting$display_dates,
-                           scenariolabels = config$report$formatting$scenario_labels,
-                           incl_geoids = included_geoids,
-                           geoid_len = 5,
-                           file_extension="auto")
+                                           num_files = nfiles,
+                                           scenariolabels = config$report$formatting$scenario_labels,
+                                           name_filter= config$hospitalization$parameters$p_death_names[1],
+                                           incl_geoids = included_geoids,
+                                           geoid_len = 5,
+                                           file_extension="auto") %>%
+    mutate(pdeath=config$hospitalization$parameters$p_death[1])
```

Comment:
Why are we defaulting to the first pdeath here?

Reply:
Good point -  we've specified a pdeath_default in the params at the top of the Rmd. We could then do : 
` mutate(pdeath=pdeath_default)`

Reply:
We don't specify a pdeath_default in the skeleton... maybe that is a MD report thing but it is something we could add.? I was defaulting to first pdeath because this is only loading infection data and it seemed safest to load the first index. Are there more robust options? @hrmeredith12 @jkamins7

Reply:
i think it's fine, if weird.  Is there a way to make it clearer to the user that this is what's happening (maybe a warning)?  I can see someone being confused by this.

Reply:
Added a warning about this in the data loading function